### PR TITLE
doc: update Thingy:91 GS to use Cellular Monitor

### PR DESCRIPTION
**Pull request by wiba-nordic**
Changes the Updating firmware section to use the new Cellular Monitor app, and moves the old information to the Developing with Thingy:91 page.

### CODE REVIEW

File: doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst [hunk 1]
```diff
@@ -104,6 +104,416 @@ Thingy:91 has a multimode modem, which enables it to support automatic switching
...
+      #. In the **Modem DFU** window, click :guilabel:`Write`.
+         When the update is complete, a "Completed successfully" message appears.
+
+         .. note::
+            Before trying to update the modem again, click the :guilabel:`Erase all` button. In this case, the contents of the flash memory are deleted and the applications must be reprogrammed.
+
+..
+
+.. rst-class:: numbered-step
+
```
Comment (divipillai) [inline doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst:409]:
```suggestion
```
leftover

File: doc/nrf/device_guides/working_with_nrf/nrf91/thingy91_gsg.rst [hunk 2]
```diff
@@ -63,447 +57,137 @@ Complete the following steps to prepare the Thingy:91 for setup:
...
+   The **Program Mode Firmware (Optional)** changes to the **Program Asset Tracker V2** window.
 
-            Programmer - Add file
+#. Switch off the Thingy:91.
+#. Press **SW3** while switching **SW1** to the **ON** position to enable MCUboot mode.
+#. Click :guilabel:`Program` to program the application to the Thingy:91.
+   Do not unplug or turn off the device during this process.
 
-      .. update_modem_start
+   When the process is complete, you see a success message.
```
Comment (divipillai) [inline doc/nrf/device_guides/working_with_nrf/nrf91/thingy91_gsg.rst:166]: Should we also mention the failure scenario . _Unable to program the device, please check the log._ 
and tell the user to program again?

Comment (wiba-nordic) [reply doc/nrf/device_guides/working_with_nrf/nrf91/thingy91_gsg.rst:166]: We don't mention the failure as an option for the nRF9160 DK Getting started, though I guess it might be more likely with the Thingy:91 if MCUboot isn't enabled properly.

Comment (divipillai) [reply doc/nrf/device_guides/working_with_nrf/nrf91/thingy91_gsg.rst:166]: yeah, I got the message 1-2 times when trying to enable the MCUboot while testing Thingy:91 :)

Comment (wiba-nordic) [reply doc/nrf/device_guides/working_with_nrf/nrf91/thingy91_gsg.rst:168]: Added a line about switching the Thingy:91 off and enabling MCUboot mode again.

**Review by divipillai — APPROVED** [review]

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf9160_gs.rst [hunk 3]
```diff
@@ -199,11 +201,11 @@ To activate the iBasis SIM card that comes shipped with the |DK| and add the |DK
 
       nRF Cloud - Add LTE Device page, Verify SIM Info view
 
-#. Complete the following steps in the **Activate SIM Card** view to activate your iBasis SIM card:
+4. Complete the following steps in the **Activate SIM Card** view to activate your iBasis SIM card:
```
Comment (melwee01) [inline doc/nrf/device_guides/working_with_nrf/nrf91/nrf9160_gs.rst:204]: This can be set back to a #

File: doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst [hunk 4]
```diff
@@ -104,6 +104,412 @@ Thingy:91 has a multimode modem, which enables it to support automatic switching
...
+      #. Navigate to where you extracted the firmware.
+
+      #. Open the folder :file:`img_app_bl` that contains the HEX files for flashing with a debugger.
+         See the :file:`CONTENTS.txt` file for information on which file you need.
+
+      #. Select the Connectivity bridge firmware file.
+      #. Click :guilabel:`Open`.
+      #. Scroll down in the menu on the left to the **DEVICE** section and click :guilabel:`Erase & write`.
+         The update is completed when the animation in Programmer's Device memory layout window ends.
```
Comment (melwee01) [inline doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst:300]: Bold since it's a UI element and the sentence is hard to parse otherwise.

File: doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst [hunk 5]
```diff
@@ -104,6 +104,412 @@ Thingy:91 has a multimode modem, which enables it to support automatic switching
...
+
+            Programmer - MCUboot DFU
+
+      #. In the **MCUboot DFU** window, click :guilabel:`Write`.
+         When the update is complete, a **Completed successfully** message appears.
+      #. Scroll up in the menu on the left to the **FILE** section and click :guilabel:`Clear files`.
+
+   .. group-tab:: Through external debug probe
+
+      To program the application firmware using external debug probe, complete the following steps:
```
Comment (melwee01) [inline doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst:478]:
```suggestion
      To program the application firmware using an external debug probe, complete the following steps:
```

File: doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst [hunk 6]
```diff
@@ -104,6 +104,412 @@ Thingy:91 has a multimode modem, which enables it to support automatic switching
...
+            If you are connecting over NB-IoT and your operator does not support extended Protocol Configuration Options (ePCO), select the file that has legacy Protocol Configuration Options (PCO) mode enabled.
+
+      #. Click :guilabel:`Open`.
+      #. Scroll down in the menu on the left to the **DEVICE** section and click :guilabel:`Erase & write`.
+         The update is completed when the animation in Programmer's **Device memory layout** window ends.
+
+         .. figure:: images/programmer_ext_debug_hex_write.png
+            :alt: Programming using External debug probe
+
+            Programming using External debug probe
```
Comment (melwee01) [inline doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst:509]: There are a couple instances of this capitalization. Not sure if there is a reason. I'd also prefer 'an external debug probe'

Comment (wiba-nordic) [reply doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst]: Fixed.

File: doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst (hunk 5 above)
Comment (wiba-nordic) [reply doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst]: Fixed.

File: doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst (hunk 4 above)
Comment (wiba-nordic) [reply doc/nrf/device_guides/working_with_nrf/nrf91/thingy91.rst]: Fixed in a couple of places.

File: doc/nrf/device_guides/working_with_nrf/nrf91/nrf9160_gs.rst (hunk 3 above)
Comment (wiba-nordic) [reply doc/nrf/device_guides/working_with_nrf/nrf91/nrf9160_gs.rst]: Fixed.

**Review by melwee01 — APPROVED** [review]
